src/site/markdown/index.md

```diff
@@ -37,12 +37,12 @@ be found on the [usage page](./usage.html).
 
 To tidy up the `pom.xml`, simply invoke it on your project, e.g.
 
-    mvn tidy:pom
+        mvn tidy:pom
```

Comment:
This one a little change an output:
from:
<img width="437" alt="image" src="https://github.com/mojohaus/tidy-maven-plugin/assets/3578921/22869730-dde6-4d78-ba45-fa5803f573c8">
to:
<img width="437" alt="image" src="https://github.com/mojohaus/tidy-maven-plugin/assets/3578921/01046a18-6fa0-48ff-b21f-6dcb81f49e72">